src/index.js

```diff
@@ -184,16 +189,11 @@ const App = ()=> {
             </>
         ):(
           <div>
-              <div className='navi'>
```

Comment:
classname and home are deleted, similar to previous comment 

  <div className='navi'>
            <nav>   
              <Link to='/home'>Home</Link>
              <Link to='/products'>Courses</Link>